ports/stm32/boards/stm32wb55xg.ld

```diff
@@ -9,7 +9,8 @@ MEMORY
     FLASH_APP (rx)  : ORIGIN = 0x08004000, LENGTH = 496K /* sectors 4-127 */
     FLASH_FS (r)    : ORIGIN = 0x08080000, LENGTH = 256K /* sectors 128-191 */
     RAM (xrw)       : ORIGIN = 0x20000000, LENGTH = 192K /* SRAM1 */
-    RAM2A (xrw)     : ORIGIN = 0x20030020, LENGTH = 8K /* SRAM2A */
+    RAM2A (xrw)     : ORIGIN = 0x20030000, LENGTH = 10K  /* SRAM2A */
+    RAM2B (xrw)     : ORIGIN = 0x20038000, LENGTH = 10K  /* SRAM2B */
```

Comment:
there's a gap (of 22k) betwoon RAM2A and RAM2B, is that intentional?

Reply:
It's a conservative estimate of the non-secure area of SRAM2A.

Reply:
Ok